tables/utils.py

```diff
@@ -34,7 +39,7 @@ def correct_byteorder(ptype, byteorder):
         return byteorder
 
 
-def is_idx(index):
+def is_idx(index: Union[int, np.integer, np.ndarray]) -> bool:
```

Comment:
Not sure about the type hint for the `index` argument.
This function checks whenever the input argument can work as an index or not, hence the arcument could also legitimately be something that cannot be used as an index

Reply:
Good point. I guess I was a little bit confused about a missing `else` case (which is just the last `return` statement).
I changed it to `Any` to allow everything.

Reply:
Thanks